website/docs/d/pn_application_chrome.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "ibm"
+page_title: "IBM : pn_application_chrome"
+sidebar_current: "docs-ibm-datasource-pn-application-chrome"
```

Comment:
Remove this sidebar ...And follow new docs updates..

Add sub category and an entry in 
allowed-subcategories.txt since its a new service
check the latest code of docs

Reply:
done